ui/modules/things/searchFilter.js

```diff
@@ -111,22 +109,98 @@ function onEnvironmentChanged() {
   updateFilterList();
 }
 
+function fillSearchFilterEdit(fillString) {
+  dom.searchFilterEdit.value = fillString;
+
+  checkIfFavourite();
+  const filterEditNeeded = checkAndMarkParameter();
+  if (!filterEditNeeded) {
+    ThingsSearch.searchTriggered(dom.searchFilterEdit.value);
+  }
+}
+
 /**
  * Updates the UI filterList
  */
 function updateFilterList() {
   dom.filterList.innerHTML = '';
   Utils.addDropDownEntries(dom.filterList, ['Favourite search filters'], true);
   Utils.addDropDownEntries(dom.filterList, Environments.current().filterList ?? []);
-  Utils.addDropDownEntries(dom.filterList, ['Field search filters'], true);
-  Utils.addDropDownEntries(dom.filterList, (Environments.current().fieldList ?? [])
-      .map((f) => `eq(${f.path},${FILTER_PLACEHOLDER})`));
   Utils.addDropDownEntries(dom.filterList, ['Example search filters'], true);
   Utils.addDropDownEntries(dom.filterList, filterExamples);
   Utils.addDropDownEntries(dom.filterList, ['Recent search filters'], true);
   Utils.addDropDownEntries(dom.filterList, filterHistory);
 }
 
+async function createFilterList(query) {
+  const date24h = new Date();
+  const date1h = new Date();
+  const date1m = new Date();
+  date24h.setDate(date24h.getDate() - 1);
+  date1h.setHours(date1h.getHours() - 1);
+  date1m.setMinutes(date1m.getMinutes() -1);
+
+  return [
+    {
+      label: 'Created since 1m',
+      rql: `gt(_created,"${date1m.toISOString()}")`,
+      group: 'Time',
+    },
+    {
+      label: 'Created since 1h',
+      rql: `gt(_created,"${date1h.toISOString()}")`,
+      group: 'Time',
+    },
+    {
+      label: 'Created since 24h',
+      rql: `gt(_created,"${date24h.toISOString()}")`,
+      group: 'Time',
+    },
+    {
+      label: 'Modified since 1m',
+      rql: `gt(_modified,"${date1m.toISOString()}")`,
+      group: 'Time',
+    },
+    {
+      label: 'Modified since 1h',
+      rql: `gt(_modified,"${date1h.toISOString()}")`,
+      group: 'Time',
+    },
+    {
+      label: 'Modified since 24h',
+      rql: `gt(_modified,"${date24h.toISOString()}")`,
+      group: 'Time',
+    },
+    {
+      label: `thingId = ${FILTER_PLACEHOLDER}`,
+      rql: `eq(thingId,"${FILTER_PLACEHOLDER}")`,
+      group: 'ThingId',
+    },
+    {
+      label: `thingId ~ ${FILTER_PLACEHOLDER}`,
+      rql: `like(thingId,"${FILTER_PLACEHOLDER}*")`,
+      group: 'ThingId',
+    },
+    {
+      label: `exists ${FILTER_PLACEHOLDER}`,
+      rql: `exists(${FILTER_PLACEHOLDER})`,
+      group: 'Other',
+    },
+    ...(Environments.current().filterList ?? []).map((f) => ({label: f, rql: f, group: 'Favourite'})),
```

Comment:
I think we mainly use American English in docs and UI .. so please change to "Favorite"